pkg/config/consts.go

```diff
@@ -43,13 +43,13 @@ const (
 	// ConsensusMaxStep consensus max step number.
 	ConsensusMaxStep = uint8(213)
 
-	// ConsensusMaxCommitteeSize represents the maximum size of the committee in
+	// ConsensusCommitteeSize represents the maximum size of the committee in
 	// 1st_Reduction, 2th_Reduction and Agreement phases.
-	ConsensusMaxCommitteeSize = 64
+	ConsensusCommitteeSize = 64
 
-	// ConsensusMaxCommitteeSize represents the maximum size of the committee in
+	// ConsensusCommitteeSize represents the maximum size of the committee in
```

Comment:
This should be `ConsensusSelectionCommitteeSize` in the comment (seems like an old typo).

Also remove `maximum size of `.